framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss

```diff
@@ -11,8 +15,7 @@
   flex-grow: 1;
   list-style: none;
   margin: 0;
-  padding-left: 0;
-  transition: 0.2s;
```

Comment:
Ok, no prob. DT don't like animations anyway, especially when they don't follow the governance that is not discussed yet about that. A small animation in the current menu of info site was not looked well as well, so good to no animate, it's safer.